### PR TITLE
feat: add version param to plugin script

### PR DESCRIPTION
**Pull request by sergsadovyi**

### CODE REVIEW

**Review by designerbrent — CHANGES_REQUESTED** [review]
I'm not seeing any change currently, as it is still pulling this URL: `http://alpswp3.test:8888/wordpress5/wp-content/plugins/alps-gutenberg-blocks/dist/blocks.build.js?ver=1`

2020-07-27 at 8 55 AM

**Comment by sergsadovyi** [comment]
Found the duplicated script enqueue

File: src/init.php [hunk 1]
```diff
@@ -421,13 +421,6 @@ function alps_gutenberg_blocks_render_block_latest_post($attributes) {
  * Registers the `core/latest-posts` block on server.
  */
 function register_block_alps_latest_posts() {
-	wp_enqueue_script(
-		'gutenberg-blocks-block-js', // Handle.
-		plugins_url('/dist/blocks.build.js', dirname(__FILE__)), // Block.build.js: We register the block here. Built with Webpack.
-		array('wp-blocks', 'wp-i18n', 'wp-element', 'wp-editor', 'wp-data'), // Dependencies, defined above.
-		// filemtime(plugin_dir_path(__DIR__) . 'dist/blocks.build.js'), // Version: filemtime — Gets file modification time.
-		true // Enqueue the script in the footer.
-	);
```
Comment (designerbrent) [inline src/init.php:430]: This this function out, the edit pages crash. 

2020-07-27 at 6 21 PM

File: src/init.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ function gutenberg_blocks_editor_assets() {
 		'gutenberg-blocks-block-js', // Handle.
 		plugins_url('/dist/blocks.build.js', dirname(__FILE__)), // Block.build.js: We register the block here. Built with Webpack.
 		array('wp-blocks', 'wp-i18n', 'wp-element', 'wp-editor', 'wp-data'), // Dependencies, defined above.
-		// filemtime(plugin_dir_path(__DIR__) . 'dist/blocks.build.js'), // Version: filemtime — Gets file modification time.
+        ALPS_GUTENBERG_VERSION,
```
Comment (designerbrent) [inline src/init.php:51]: Can you add this to the CSS scripts below?

**Review by designerbrent — CHANGES_REQUESTED** [review]

**Comment by designerbrent** [comment]
Closing this as it will be covered in #36.

**Pull request closed without merging**